freqtrade/optimize/backtesting.py

```diff
@@ -358,6 +358,94 @@ def _get_close_rate(self, sell_row: Tuple, trade: LocalTrade, sell: SellCheckTup
         Get close rate for backtesting result
         """
         # Special handling if high or low hit STOP_LOSS or ROI
+        is_short = trade.is_short or False
+        if is_short:
+            return self._get_short_close_rate(sell_row, trade, sell, trade_dur)
+        else:
+            return self._get_long_close_rate(sell_row, trade, sell, trade_dur)
+
+    def _get_short_close_rate(self, sell_row: Tuple, trade: LocalTrade, sell: SellCheckTuple,
```

Comment:
I'm not entirely sure i'm happy with the split between short and long close rate.

It may seem to simplify the logic - but in reality, it'll introduce a (quite high) risk of the 2 slowly drifting apart over time (the code is not next to each other - if people attempt to fix one bug, they're less likely to go someplace "completely different" (different method in this case) to try and fix the other problem.


Assuming you've been forced into this for "flake8 complexity" reasons - how about splitting the method by sell_type instead?